src/servidor/CadastroServidorPage.tsx

```
@@ -68,37 +71,75 @@ export function CadastroServidorPage() {
-                toast.success('Servidor cadastrado com sucesso!')
-                void navigate('/')
-            })
-            .catch((error: AxiosError<ErrorResponse>) => {
-                const errorResponse = error.response?.data;
-                if (Array.isArray(errorResponse?.validationErrors) && errorResponse.validationErrors.length > 0) {
-                    toast.error(errorResponse.validationErrors[0]);
-                } else {
-                    toast.error('Erro ao cadastrar servidor: ' + (errorResponse?.message || error.message));
-                }
-            })
+        console.log('Payload do servidor:', payload)
```

ESLint

[error] Delete `····`

(prettier/prettier)